packages/sdk/src/evm/constants/erc1155-features.ts

```diff
@@ -24,40 +25,40 @@ export const FEATURE_EDITION_BURNABLE = {
 } as const;
 
 export const FEATURE_EDITION_CLAIM_CONDITIONS_V1 = {
-  name: "ERC1155ClaimConditionsV1",
+  name: "ERC1155DropSinglePhaseV1",
   namespace: "edition.drop.claim",
   docLinks: {
     sdk: "sdk.erc1155claimable",
     contracts: "DropSinglePhase1155",
   },
-  abis: [Erc1155Abi, ILazyMintAbi, IDropSinglePhase1155_V1],
+  abis: [Erc1155Abi, IDropSinglePhase1155_V1],
   features: {},
 } as const;
 
 export const FEATURE_EDITION_CLAIM_CONDITIONS_V2 = {
-  name: "ERC1155ClaimConditionsV2",
+  name: "ERC1155DropSinglePhase",
   namespace: "edition.drop.claim",
   docLinks: {
     sdk: "sdk.erc1155claimable",
     contracts: "DropSinglePhase1155",
   },
-  abis: [Erc1155Abi, ILazyMintAbi, IDropSinglePhase1155],
+  abis: [Erc1155Abi, IDropSinglePhase1155],
```

Comment:
removing LazyMint Abi detection should be safe though, that can be done in another PR on its own and can easily be merged